config/v1/0000_10_config-operator_01_infrastructure-Default.crd.yaml

```diff
@@ -593,6 +593,9 @@ spec:
                     external:
                       description: External contains settings specific to the generic External infrastructure provider.
                       type: object
+                      x-kubernetes-validations:
+                        - message: cloudControllerManager added or removed once set
+                          rule: has(self.cloudControllerManager) == has(oldSelf.cloudControllerManager)
```

Comment:
since we are putting this behind a feature gate for testing, it seems like removing the validation for the moment would be accepable.

Reply:
If you can wait until the end of next week, this issue has already been raise and I intend to work during shift week to find a solution for this. My intention is to allow CEL validations to be gated by a featureset, as has been done for fields themselves

Reply:
Spoke with Mike about this, I think we can drop this particular validation for now and re-introduce later.
It is safe to introduce later since it is a transitional validation, ie, no matter the data that exists in etcd, it won't cause an invalid data to be stored, also, as this is tech preview, we can adjust before we make it GA